File: 백준/Bronze/2475. 검증수/검증수.c
#include <stdio.h>

int main()
{
    int n, i, sum = 0;
    for (i = 0; i < 5; i++){
        scanf("%d", &n);
        sum += n * n;
    }
    printf("%d", sum % 10);
    return 0;
}

File: 백준/Bronze/2475. 검증수/검증수.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int sum = 0;
        for(int i = 0; i < 5; i++) {
            sum += Math.pow(Integer.parseInt(st.nextToken()), 2);
        }
        System.out.println(sum % 10);
    }
}